.readthedocs.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+sphinx:
+  fail_on_warning: true
+
+formats:
+  - pdf
+  - epub
+
+python:
+  version: 3.8
+  system_packages: False
+  install:
+    - requirements: docs/rtd-requirements.txt
```

Comment:
I would prefer to have all the dependencies defined in a single file.

Right now, all the deps are defined in src/twisted/python/_setup.py ...

So if you think that we need a separate list of `rtd` deps, it should be defined there.

But I am +1 on just using the current `dev` list of dependencies